packages/data-context/src/actions/AuthActions.ts

```diff
@@ -3,12 +3,14 @@ import type { AuthenticatedUserShape, AuthStateShape } from '../data'
 
 export interface AuthApiShape {
   getUser(): Promise<Partial<AuthenticatedUserShape>>
-  logIn(onMessage: (message: AuthStateShape) => void): Promise<AuthenticatedUserShape>
+  logIn(onMessage: (message: AuthStateShape) => void, utmCode: string): Promise<AuthenticatedUserShape>
   logOut(): Promise<void>
   resetAuthState(): void
 }
 
 export class AuthActions {
+  private cancelActiveLogin: (() => void) | null = null
+
```

Comment:
I noticed that this was being set on `coreData`, but never actually used outside of this class... so I made it a local private variable. Any reason I'm missing to not do this? There's nothing about `Actions` not holding local state in the `README`, but I noticed none of them seem to do it.

Or, well, they do have local state, but they put it in `coreData` :stuck_out_tongue:

Reply:
Yeah it was generally because we don't create a new `DataContext`, but we do reset/create a new `coreData` between tests, so it's cleaner to keep it there from a testing perspective. 

We originally wanted to keep state centralized in `coreData` so that we could eventually make it immutable to better track updates to it for debugging, but i'm not sure if we've been consistent in that regard.

I don't feel strongly about it either way, though we've started making `private` members [actually "private"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) elsewhere, so if you want to keep it here, want to change it to `#cancelActiveLogin`?

Reply:
Kept it in as `#cancelActiveLogin`: 7572f7811f